airbyte-integrations/connectors/source-pokeapi/README.md

```diff
@@ -1,103 +1,65 @@
-# Pokeapi Source
+# Pokeapi source connector
```

Comment:
I cobbled the updated README template together from various places, so it could probably use a solid proofread before we move forward, as it will be recycled for all migrated connectors. Now's the time to get it right!